### ISSUE TITLE
ValueError: Cannot set a frame with no defined index and a value that cannot be converted to a Series

### ISSUE THREAD

**Issue by hcdenbakker** [gem-pasteur/Integron_Finder#16]
I got the following error trying to use your software:
command: 
`integron_finder Enteritidis_P125109.fasta --local_max --func_annot --linear`

terminal output:
```
-------------------------------------
PRODIGAL v2.6.2 [February, 2015]         
Univ of Tenn / Oak Ridge National Lab
Doug Hyatt, Loren Hauser, et al.     
-------------------------------------
Request:  Single Genome, Phase:  Training
Reading in the sequence(s) to train...4685848 bp seq created, 52.17 pct GC
Locating all potential starts and stops...227916 nodes
Looking for GC bias in different frames...frame bias scores: 1.14 0.17 1.69
Building initial set of genes to train from...done!
Creating coding model and scoring nodes...done!
Examining upstream regions and training starts...done!
-------------------------------------
Request:  Single Genome, Phase:  Gene Finding
Finding genes in sequence #1 (4685848 bp)...done!

>>> Starting Default search ... :
>>> Default search done... : 

Traceback (most recent call last):
  File "/usr/local/bin/integron_finder", line 1667, in <module>
    phageI_file)
  File "/usr/local/bin/integron_finder", line 577, in find_integron
    size_min_attc=min_attc_size)
  File "/usr/local/bin/integron_finder", line 1228, in read_infernal
    df.loc[idx, "pos_beg"] = df.loc[idx].apply(lambda x: x["pos_end_tmp"] - (length_cm - x["cm_fin"]), axis=1)
  File "/usr/local/lib/python2.7/dist-packages/pandas/core/indexing.py", line 128, in __setitem__
    self._setitem_with_indexer(indexer, value)
  File "/usr/local/lib/python2.7/dist-packages/pandas/core/indexing.py", line 280, in _setitem_with_indexer
    self.obj[key] = value
  File "/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py", line 2348, in __setitem__
    self._set_item(key, value)
  File "/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py", line 2413, in _set_item
    self._ensure_valid_index(value)
  File "/usr/local/lib/python2.7/dist-packages/pandas/core/frame.py", line 2395, in _ensure_valid_index
    raise ValueError('Cannot set a frame with no defined index '
ValueError: Cannot set a frame with no defined index and a value that cannot be converted to a Series

```
I received the same error on the mobyle webserver.

Best regards,
-Henk den Bakker

**Comment by jeanrjc** [comment]
Hi Henk,

Can you provide me the sequence so I try to reproduce the error ? Can you tell me your version of pandas too ?

Best,
Jean

**Comment by jeanrjc** [comment]
I found your sequence and were able to reproduce the error, I investigate.

**Comment by jeanrjc** [comment]
Thanks for your report, there was a small bug which is now fix in version 1.5 which you can download here:
https://github.com/gem-pasteur/Integron_Finder/releases/tag/v1.5

beware, I update some function to be up to date with pandas, and pandas version 0.18 is now needed.
